java/clients/src/main/java/sleeper/clients/admin/IngestStatusReportScreen.java

```diff
@@ -78,7 +77,7 @@ public void chooseArgsAndPrint(String instanceId) throws InterruptedException {
     }
 
     private void chooseArgsForIngestJobStatusReport(InstanceProperties properties) throws InterruptedException {
-        Optional<TableProperties> tableOpt = tableSelectHelper.chooseTableOrReturnToMain(properties.get(ID));
+        Optional<TableProperties> tableOpt = tableSelectHelper.chooseTableOrReturnToMain(properties);
```

Comment:
We don't need to load the table properties here.

Reply:
I can see there's a benefit to this right now in that if the table doesn't exist you'll get an error message. We could make that error message clearer to explain when the table doesn't exist. Whether we're loading the properties or not, we could separate the case where the table doesn't exist from the case where some other failure occurred.